@here/harp-map-controls/lib/MapControls.ts

```diff
@@ -147,7 +147,7 @@ export class MapControls extends THREE.EventDispatcher {
      * If defined, restricts all camera movements (pan, zoom, orbit) triggered from these map
      * controls so that camera position remains within the specified bounding box.
      */
-    bounds?: geoUtils.GeoBox;
+    bounds?: GeoBox;
```

Comment:
this was the offending line for api-extractor. It was the first time a MapControls member (appearing in MapControls.d.ts) was using a type imported with wildcard.